pull-dep.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
It looks like this file was unintentionally added to the PR.

Reply:
thanks. It's fixed now.